validator/Cargo.toml

```diff
@@ -50,6 +50,7 @@ solana-net-utils = { workspace = true }
 solana-perf = { workspace = true }
 solana-poh = { workspace = true }
 solana-program-runtime = { workspace = true }
+solana-rayon-threadlimit = { workspace = true }
```

Comment:
This crates goes away long-term, just bubbling this up so that this PR does not alter existing behavior